framework/db/Query.php

```diff
@@ -248,6 +248,19 @@ public function one($db = null)
     }
 
     /**
+     * Executes the query and returns a first single row of result.
+     * @param Connection $db the database connection used to generate the SQL statement.
+     * If this parameter is not given, the `db` application component will be used.
+     * @return array|boolean the first row (in terms of an array) of the query result. False is returned if the query
```

Comment:
Query returns array, not object.

And...
This is copy of phpdoc from one() method.